### PR TITLE
Support `roundingMode` for InputNumber

### LINKED ISSUE #5528: InputNumber: Support `roundingMode` for the InputNumber
### Discussed in https://github.com/orgs/primefaces/discussions/611

<div type='discussions-op-text'>

<sup>Originally posted by **trandaison** December  4, 2023</sup>
Since `InputNumber` is using `Intl.NumberFormat` for formating the value.
There are some `options` are not supported, such as [`roundingMode`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingmode).

How about adding the `roundingMode` props for `InputNumber`?</div>

### PR DESCRIPTION
**Pull request by trandaison**
### Feature Requests
Support [`roundingMode`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#roundingmode) for the `InputNumber` component.

https://github.com/orgs/primefaces/discussions/611

Fix #5528

### CODE REVIEW

**Comment by melloware** [comment]
Please open a GitHub issue on PrimeReact so we can link this ticket to it.

**Review by melloware — CHANGES_REQUESTED** [review]
See comment

**Comment by melloware** [comment]
Nevermind I ported over your discussion to an issue ticket

**Review by melloware — APPROVED** [review]

File: components/lib/inputnumber/InputNumber.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ export const InputNumber = React.memo(
                 currencyDisplay: props.currencyDisplay,
                 useGrouping: props.useGrouping,
                 minimumFractionDigits: props.minFractionDigits,
-                maximumFractionDigits: props.maxFractionDigits
+                maximumFractionDigits: props.maxFractionDigits,
+                roundingMode: props.roundingMode,
```
Comment (melloware) [inline components/lib/inputnumber/InputNumber.js:62]: Don't you need to add `roundingMode` to `InputNumberBase.js` as well?

**Review by melloware — CHANGES_REQUESTED** [review]

File: components/lib/inputnumber/InputNumber.js (hunk 1 above)
Comment (trandaison) [reply components/lib/inputnumber/InputNumber.js:62]: Thanks! I added default value for `roundingMode` in `InputNumberBase`.

**Comment by melloware** [comment]
Looks like prettier failed can you run `npm run format` to format your code?

**Review by melloware — APPROVED** [review]

**Pull request merged**